databases/week1/homeWork.sql

```diff
@@ -0,0 +1,92 @@
+create DATABASE hyf_db;
+use hyf_db;
+
+CREATE TABLE tasks (
```

Comment:
You should use the database schema (meaning the tables) and data from the [lesson](https://github.com/HackYourFuture-CPH/databases/tree/main/lesson1). An important part of the homework is to practice joining tables, and the way you defined the `tasks` table, there is no need for `JOIN` commands.